packages/react-composites/tests/browser/chat/fake-adapter/FileSharing.test.ts

```diff
@@ -292,3 +306,12 @@ test.describe('Filesharing Edit Message', async () => {
     ).toMatchSnapshot('filesharing-file-upload-card-while-editing-message.png');
   });
 });
+
+async function waitIfTestingOnMobileLandscape(workerInfo, page): Promise<void> {
+  if (
+    workerInfo.project.name.toLowerCase().includes('mobile') &&
+    workerInfo.project.name.toLowerCase().includes('landscape')
+  ) {
+    await page.waitForTimeout(1000);
```

Comment:
Does the 1 second here guarantee that we don't see the scroll bar?

Reply:
Not absolutely. But I thought it would suffice. I was thinking of something in the DOM tree or some kind of javascript event to wait for but I could not think of anything. Do you have any ideas?

Reply:
Another option is to hide the chat container scrollbar with css, but we'd want to do that _only_ for mobile landscape and only the message thread component

Reply:
Actually see other thread, I really feel this is an issue with filesharing that needs fixed